resources/service/proxy.py

```diff
@@ -0,0 +1,163 @@
+import socket
+import requests
+import threading
+import http
+from http import HTTPStatus
+from http.server import ThreadingHTTPServer, BaseHTTPRequestHandler
+from io import BytesIO
+
+from urllib.parse import urlparse, parse_qs, urlencode
```

Comment:
I'm pretty sure urllib.parse doesn't exists in Python 2 https://stackoverflow.com/questions/50638366/url-parse-python2-7-equivalent